charts/nextcloud/templates/deployment.yaml

```diff
@@ -329,17 +329,18 @@ spec:
       {{- if .Values.nextcloud.extraVolumes }}
 {{ toYaml .Values.nextcloud.extraVolumes | indent 6 }}
       {{- end }}
+      # this is the securityContext for ALL containers in the nextcloud pod
       securityContext:
       {{- if .Values.nextcloud.podSecurityContext }}
         {{- with .Values.nextcloud.podSecurityContext }}
           {{- toYaml . | nindent 8 }}
         {{- end }}
       {{- else }}
-        {{- if .Values.nginx.enabled }}
-        # Will mount configuration files as www-data (id: 82) for nextcloud
+        {{- if regexmatch "alpine$" .Values.image.flavor || if regexmatch "alpine$" Values.image.tag }}
```

Comment:
Not sure if the regex should only match images that end with `alpine`. Tbh it's impossible to tell if an image is really alpine based, because it doesn't need to have alpine in the name. Maybe adding a switch to tell the chart if the image is alpine would be better? I'm not sure

Reply:
You are right (with switch I meant just having a boolean value somewhere).